app/serializers/rule_serializer.rb

```diff
@@ -5,7 +5,9 @@ class RuleSerializer < ApplicationSerializer
   attributes :ref_id, :remediation_issue_id, :title, :rationale, :description,
              :severity, :slug
   belongs_to :benchmark
-  has_many :profiles
+  has_many :profiles do |rule|
+    Pundit.policy_scope(User.current, rule.profiles)
```

Comment:
Would this work on user-less contexts, e.g. on SSG import?

Reply:
Is the SSG import using the serializer?

Reply:
:facepalm: I mistook it for an ActiveRecord relationship... All good.